lib/layer.js

```diff
@@ -48,6 +48,31 @@ function Layer(path, options, fn) {
 }
 
 /**
+ * Handle any request for the layer.
+ *
+ * @api private
+ */
+
+Layer.prototype._handle_any = function _handle_any(/* ...args, next */) {
+  var self = this
+  var next = arguments[arguments.length - 1]
+  var handle = this.handle
+  var result
+  var hasNextBeenCalled = function () { return next._currentLayer === self }
+
+  try {
+    result = handle.apply(undefined, arguments)
```

Comment:
@dougwilson If I understand, this will still function the same as this is already under strict mode. Also, `call` and `apply` actually work the same here (when you pass `undefined` in non-strict mode it would be global, but this file is strict).